pipeline/backend/kubernetes/pod_test.go

```diff
@@ -51,7 +51,7 @@ func TestStepToPodName(t *testing.T) {
 	assert.EqualValues(t, "wp-01he8bebctabr3kg", name)
 	name, err = stepToPodName(&types.Step{UUID: "01he8bebctabr3kg", Name: "postgres", Type: types.StepTypeService})
 	assert.NoError(t, err)
-	assert.EqualValues(t, "postgres", name)
+	assert.EqualValues(t, "postgres-01he8bebctabr3kg", name)
```

Comment:
I would make it suffix as it was before. Something like `wp-01he8bebctabr3kg-postgres` / `wp-01he8bebctabr3kg-svc-postgres` / `wp-01he8bebctabr3kg-service-postgres`.